drafting/conda-template.py

```diff
@@ -0,0 +1,85 @@
+# This script reads in a yaml file (qiime2 recipe)
```

Comment:
This file should have a hash-pling, and an [`if __name__ == '__main__'`](https://stackoverflow.com/a/419185) guard.